system/config/templates/features/aca_individual_market/financial_assistance.yml

```diff
@@ -145,6 +145,8 @@ registry:
       item: "curam"
     - key: :non_magi_transfer
       is_enabled: <%= ENV['NON_MAGI_TRANSFER_IS_ENABLED'] || false %>
+    - key: :block_renewal_application_transfers
+      is_enabled: <%= ENV['BLOCK_RENEWAL_ATP_IS_ENABLED'] || false %>
```

Comment:
would `ATP_RENEWAL_IS_ENABLED` be more readable here?

true = atp renewal is on
false = atp renewal is off


with `BLOCK_` we have to make the mental shift to:
true = atp renewal is off
false = atp renewal is on

Reply:
changes ENV name